data/json/itemgroups/Weapons_Mods_Ammo/gunmod.json

```diff
@@ -21,7 +21,9 @@
       [ "cheek_pad", 25 ],
       [ "rail_laser_sight", 40 ],
       [ "red_dot_sight", 70 ],
-      [ "choke", 50 ]
+      [ "choke", 50 ],
+      [ "big_brass_catcher", 20 ],
+      [ "offset_grip", 25 ]
```

Comment:
I don't think these should be that common. Granted, there's no good evidence on how common a gunmod is, but this seems like a pretty niche attachment that's not very common, I'd set it around 5.